### DISCUSSION TITLE
Missing policies for certain traffic selectors

### DISCUSSION THREAD

**Discussion by gr4yg00s3** [strongswan/strongswan#957] [General]
Hello, 

I'm looking into some weird behavior I am facing. Having configured swanctl.conf with the appropriate config for a site-to-site connection, sometimes I will randomly see traffic to/from a particular participating subnet is dropped. When I check `ip xfrm policy` to see if the policy for said subnet exists appropriately, I happen to find it missing. This was the first indication that something is wrong. On being unable to understand why this policy is missing, I restarted `charon` and it seems like traffic to/from said subnet is active again.
I enabled KNL logs at level 2, but I don't see any error. Is there anything I can look at particularly or is there something else I can try. Would appreciate any help with this.

**Comment by tobiasbrunner** [comment]
Other than studying the logs (kernel logs maybe too), no idea.
